ksql-engine/src/main/java/io/confluent/ksql/internal/KsqlEngineMetrics.java

```diff
@@ -143,62 +147,49 @@ private void recordErrorRate(final double value) {
-                + "Alternately, a consumed messages may not have been produced, hence "
-                + "being effectively dropped. Such messages would also be counted "
-                + "toward the error rate."),
-        new Value());
-    return sensor;
+    final String metricName = "error-rate";
+    final String description =
+        "The number of messages which were consumed but not processed. "
+        + "Messages may not be processed if, for instance, the message "
+        + "contents could not be deserialized due to an incompatible schema. "
+        + "Alternately, a consumed messages may not have been produced, hence "
+        + "being effectively dropped. Such messages would also be counted "
+        + "toward the error rate.";
+    return createSensor(metrics, metricName, description, Value::new);
   }
 
   private Sensor configureMessagesOut(final Metrics metrics) {
-    final Sensor sensor = createSensor(metrics, metricGroupName + "-messages-produced");
-    sensor.add(
-        metrics.metricName(ksqlServiceId + "messages-produced-per-sec", this.metricGroupName,
-            "The number of messages produced per second across all queries"),
-        new Value());
-
-    return sensor;
+    final String metricName = "messages-produced-per-sec";
+    final String description = "The number of messages produced per second across all queries";
+    return createSensor(metrics, metricName, description, Value::new);
   }
 
   private Sensor configureMessagesIn(final Metrics metrics) {
-    final Sensor sensor = createSensor(metrics, metricGroupName + "-messages-consumed");
-    sensor.add(
-        metrics.metricName(ksqlServiceId + "messages-consumed-per-sec", this.metricGroupName,
-            "The number of messages consumed per second across all queries"),
-        new Value());
-    return sensor;
+    final String metricName = "messages-consumed-per-sec";
+    final String description = "The number of messages consumed per second across all queries";
+    return createSensor(metrics, metricName, description, Value::new);
   }
 
   private Sensor configureTotalMessagesIn(final Metrics metrics) {
-    final Sensor sensor = createSensor(metrics, metricGroupName + "-total-messages-consumed");
-    sensor.add(
-        metrics.metricName(ksqlServiceId + "messages-consumed-total", this.metricGroupName,
-            "The total number of messages consumed across all queries"),
-        new Value());
-    return sensor;
+    final String metricName = "total-messages-consumed";
+    final String description = "The total number of messages consumed across all queries";
+    return createSensor(metrics, metricName, description, Value::new);
   }
 
   private Sensor configureTotalBytesIn(final Metrics metrics) {
-    final Sensor sensor = createSensor(metrics, metricGroupName + "-total-bytes-consumed");
-    sensor.add(
-        metrics.metricName(ksqlServiceId + "bytes-consumed-total", this.metricGroupName,
```

Comment:
Are we worried about changing existing metric names like this one? Same question for `configureTotalMessagesIn()` above.

Reply:
Yes, good catch!